src/components/TheSidebar.vue

```diff
@@ -129,7 +129,11 @@ onMounted(() => {
               <router-link
                 :to="{ name: 'spaceProposals', params: { key: element } }"
               >
-                <Token :space="spaces[element]" symbolIndex="space" size="44" />
+                <Token
+                  :space="explore.spaces.find(s => s.id === element)"
```

Comment:
Can this be undefined? if yes, then what should be rendered?

Reply:
I don't see how it could be undefined.

Reply:
Why not using `explore.spaces[element]` ?

Reply:
Same reason as here https://github.com/snapshot-labs/snapshot/pull/671#discussion_r712965632